pyanaconda/core/configuration/anaconda.py

```diff
@@ -0,0 +1,225 @@
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+#  Author(s):  Vendula Poncova <vponcova@redhat.com>
+#
+import os
+from abc import ABC
+
+from pyanaconda.core.constants import ANACONDA_CONFIG_TMP, ANACONDA_CONFIG_DIR
+from pyanaconda.core.configuration.base import create_parser, read_config, write_config, \
+    get_option, set_option
+
+__all__ = ["conf", "AnacondaConfiguration"]
+
+
+class Section(ABC):
+    """A base class for representation of a configuration section."""
+
+    def __init__(self, section_name, parser):
+        self._section_name = section_name
+        self._parser = parser
+
+    def _get_option(self, option_name, converter=None):
+        """Get a converted value of the option.
+
+        :param option_name: an option name
+        :param converter: a function or None
+        :return: a converted value
+        """
+        return get_option(self._parser, self._section_name, option_name, converter)
+
+    def _set_option(self, option_name, value):
+        """Set the option.
+
+        :param option_name: an option name
+        :param value: an option value
+        """
+        set_option(self._parser, self._section_name, option_name, value)
+
+
+class AnacondaSection(Section):
+    """The Anaconda section."""
+
+    @property
+    def debug(self):
+        """Run Anaconda in the debugging mode."""
+        return self._get_option("debug", bool)
+
+
+class ServicesSection(Section):
+    """The Services section."""
+
+    @property
+    def selinux(self):
+        """Enable SELinux usage in the installed system.
+
+        Valid values:
+
+         -1  The value is not set.
+          0  SELinux is disabled.
```

Comment:
I think it might be good to mention the SELinux terminology there rather than enabled/disabled:
disabled = permissive
enabled = enforcing

Reply:
Fixed.